playbooks/tests/unit_tests.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Unit tests for Linchpin
+  hosts: localhost
+  gather_facts: False
+  tasks:
+          - name: debug
+            debug:
+              msg: "Disabling linchpin tests until container is fixed. Refer Travis ci for unit tests and flake8 tests"
+          - name: Install Test Reqs
+            pip:
+              name: .[tests]
+              virtualenv: /tmp/linchpin
+              args:
```

Comment:
how did you restore this commit??  did you revert?  line 13 doesn't look like it's indented correctly

Reply:
It was commented before the unit-tests were completely removed. I uncommented them to start tests again.
Indentation went off while I uncommented the parts of the playbook